### PR TITLE
ROC-7123 Claimant receives notification for default CCJ.

### PR DESCRIPTION
**Pull request by karthikkannankainos**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-7123

### Change description ###
Notify claimant about default CCJ after 10 days of raising CCJ.

### CODE REVIEW

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/test/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJobTest.java  1
- src/main/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJob.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=4820006&bid=16024603)

File: src/main/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJob.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        this.ccjNotificationService = ccjNotificationService;
+    }
+
+    @Override
+    public void execute(JobExecutionContext jobExecutionContext) throws JobExecutionException {
+        try {
+            User anonymousCaseWorker = userService.authenticateAnonymousCaseWorker();
+            List<Claim> claimsWithCCJ = caseSearchApi.getClaimsWithDefaultCCJ(anonymousCaseWorker,
+                LocalDate.now().minusDays(claimantCCJReminderDays));
+            claimsWithCCJ.forEach(claim -> ccjNotificationService.notifyClaimantAboutCCJReminder(claim));
```
Comment (daveoh) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJob.java]: can be simplified to 
```suggestion
        caseSearchApi.getClaimsWithDefaultCCJ(anonymousCaseWorker, LocalDate.now().minusDays(claimantCCJReminderDays)
       .forEach(claim -> ccjNotificationService.notifyClaimantAboutCCJReminder(claim));
```

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJob.java  3
- src/test/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJobTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=4820006&bid=16024603)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/test/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJobTest.java  1
- src/main/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJob.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=4820006&bid=16024603)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJob.java  3
- src/test/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJobTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=4820006&bid=16024603)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/test/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJobTest.java  1
- src/main/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJob.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=4820006&bid=16024603)

**Comment by karthikkannankainos** [comment]
The PR build was green https://build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_CMC%2Fcmc-claim-store/detail/PR-1379/3/pipeline but still the PR didnt get call from jenkins.

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJob.java  3
- src/test/java/uk/gov/hmcts/cmc/claimstore/jobs/cron/NotifyClaimantDefaultCCJJobTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=4820006&bid=16024603)

**Pull request merged**